src/sharding/ShardingManager.js

```diff
@@ -34,19 +34,20 @@ class ShardingManager extends EventEmitter {
     }, options);
 
     /**
-     * Path to the shard script file
-     * @type {string}
-     */
+    * Path to the shard script file
+    * @type {string}
+    */
     this.file = file;
     if (!file) throw new Error('CLIENT_INVALID_OPTION', 'File', 'specified.');
     if (!path.isAbsolute(file)) this.file = path.resolve(process.cwd(), file);
     const stats = fs.statSync(this.file);
     if (!stats.isFile()) throw new Error('CLIENT_INVALID_OPTION', 'File', 'a file');
 
     /**
-     * Amount of shards that this manager is going to spawn
-     * @type {number|string}
-     */
+    * Amount of shards that this manager is going to spawn
+    * @type {number|string}
+    */
```

Comment:
This JSDoc would hit the `autoShards` property. Either move the value below `totalShards` or above the JSDoc.

Reply:
Yes, I didnt notice that, I fixed that as well. Also added a JSDoc Set for the new property.